packages/twenty-front/src/modules/object-record/record-board/record-board-column/components/RecordBoardColumnNewButton.tsx

```diff
@@ -21,13 +24,53 @@ const StyledButton = styled.button`
   }
 `;
 
-export const RecordBoardColumnNewButton = () => {
+export const RecordBoardColumnNewButton = ({
+  columnId,
+}: {
+  columnId: string;
+}) => {
   const theme = useTheme();
-  const { handleAddNewCardClick } = useAddNewCard('last');
+  const { visibleFieldDefinitionsState } = useRecordBoardStates();
+
+  const visibleFieldDefinitions = useRecoilValue(
+    visibleFieldDefinitionsState(),
+  );
+  const labelIdentifierField = visibleFieldDefinitions.find(
+    (field) => field.isLabelIdentifier,
+  );
+
+  const newRecord = useRecoilValue(
+    recordBoardNewRecordByColumnIdSelector({
+      familyKey: columnId,
+      scopeId: columnId,
+    }),
+  );
+
+  const { handleAddNewCardClick, handleCreateSuccess } = useAddNewCard();
+
+  const handleNewButtonClick = () => {
+    handleAddNewCardClick(
+      labelIdentifierField?.label ?? '',
+      '',
+      'last',
+      columnId,
+    );
+  };
+
+  if (newRecord.isCreating && newRecord.position === 'last') {
+    return (
+      <RecordBoardCard
+        isCreating={true}
+        onCreateSuccess={() => handleCreateSuccess('last')}
+        position="last"
+      />
+    );
+  }
+
```

Comment:
Seems like duplicate code with `packages/twenty-front/src/modules/object-record/record-board/record-board-column/components/RecordBoardColumnHeader.tsx`

Reply:
@bosiraphael Done! I had to create a new hook due to a scoping issue, as it couldn't be kept within `useAddNewCard`